### ISSUE TITLE
provide code fix for variable declarations inside foreach loop

### ISSUE THREAD

**Issue by tugberkugurlu** [DotNetAnalyzers/VarUsageAnalyzer#10]
currently, variable declarations inside foreach loop are not detected. E.g.:

``` csharp
foreach (var item in methodDeclaration.ParameterList.Parameters)
{
}
```
